src/Main.purs

```diff
@@ -195,18 +196,18 @@ runBundle cliRoot args = do
     , "      (compile-whole-program \"" <> mainWpoPath <> "\" \"" <> outPath <> "\")"
     , ")))"
     ]
-  case res of
-    Left err -> liftEffect $ Process.exit (fromMaybe 1 err.exitCode)
-    Right _ -> Console.log $ "Created " <> outPath
+  case res.exitCode of
+    Just 0 -> Console.log $ "Created " <> outPath
+    _ -> liftEffect $ Process.exit' (fromMaybe 1 res.exitCode)
 
-evalScheme :: Array String -> String -> Aff (Either ExecaError ExecaSuccess)
+evalScheme :: Array String -> String -> Aff ExecaResult
 evalScheme arguments code = do
   schemeBin <- getSchemeBinary
   spawned <- execa schemeBin arguments identity
   spawned.stdin.writeUtf8End code
   void $ liftEffect $ Stream.pipe spawned.stdout.stream Process.stdout
   void $ liftEffect $ Stream.pipe spawned.stderr.stream Process.stderr
```

Comment:
This can be `spawned.stdout.pipeToParentStdout` and `spawned.stderr.pipeToParentStderr`

Reply:
Great point